logger/logger.go

```diff
@@ -3,12 +3,19 @@ package logger
 import (
 	"github.com/Sirupsen/logrus"
 	ravenSentry "github.com/evalphobia/logrus_sentry"
+	elastic "gopkg.in/olivere/elastic.v5"
+	elogrus "gopkg.in/sohlich/elogrus.v2"
 )
 
 var (
 	environment string
 )
 
+const (
+	elasticsearchURL   = "https://search-wisegrowth-jl3kj4kwv225mzcssitu47xlqe.us-west-2.es.amazonaws.com"
```

Comment:
use environment vars!!!!! jajaja its a joke
but should use env var for other services such as core, led or ble